cache/sampleStore.js

```diff
@@ -0,0 +1,252 @@
+/**
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * ./cache/sampleStore.js
```

Comment:
some functions/variables in this file are used by most of other redis cache related files, but some are not (like eradicate/populate) when imported. Would it be better if we separate populate redis logic from redis config vars/functions ?